### ISSUE TITLE
[OUDS] [Tokens] Integration of System library v[2.1.0, 2.2.0, 2.3.0], component changelog v[1.10, 1.11], Core OUDS v[1.8, 1.9]

### ISSUE THREAD

**Issue by pylapp** [Orange-OpenSource/Orange-Boosted-Bootstrap#3221]
Tokens update integration

# System Library

### Border-width (Wireframe theme)

References updated:
- [ ] border-width-thin→core-ouds-border-width-50
 
### Size-max-width-type

References added:
- [ ] size-max-width-type-label-xlarge-mobile→core-ouds-dimension-6000
- [ ] size-max-width-type-label-xlarge-tablet→core-ouds-dimension-6000
- [ ] size-max-width-type-label-xlarge-desktop→core-ouds-dimension-7000
- [ ] size-max-width-type-label-large-mobile→core-ouds-dimension-6000
- [ ] size-max-width-type-label-large-tablet→core-ouds-dimension-6000
- [ ] size-max-width-type-label-large-desktop→core-ouds-dimension-7000
- [ ] size-max-width-type-label-medium-mobile→core-ouds-dimension-6000
- [ ] size-max-width-type-label-medium-tablet→core-ouds-dimension-6000
- [ ] size-max-width-type-label-medium-desktop→core-ouds-dimension-6000
- [ ] size-max-width-type-label-small-mobile→core-ouds-dimension-6000
- [ ] size-max-width-type-label-small-tablet→core-ouds-dimension-6000
- [ ] size-max-width-type-label-small-desktop→core-ouds-dimension-6000

### Dialog-alert

Numerous references were added to meet the needs of the component's creation → [Alert tokens repository](https://www.figma.com/design/AGpy41dOJU1kWV2GF22WJX/Navigation---Bar?node-id=16002-35707&t=OaVAEqbyIe9gzFC8-4)

The changelog file is available here:
https://www.figma.com/design/Co2t6wHMf4GB9NJVGs2Hes/-OUDS-Lib--Design-tokens?node-id=10267-1315&t=oIMR9q7leDxw9T26-4

#### References added:
- [ ] color-action-read-only-primary-light→color-repository-opacity-black-higher
- [ ] color-action-read-only-primary-dark→color-repository-opacity-white-high
- [ ] color-action-read-only-secondary-light→color-repository-opacity-black-medium-low
- [ ] color-action-read-only-secondary-dark→color-repository-opacity-white-medium-low
 
#### References added:
- [ ] color-border-minimal-light→color-repository-opacity-black-lowest
- [ ] color-border-minimal-dark→color-repository-opacity-white-lowest

### Navigation-bar

Numerous references were added to meet the needs of the component's creation → [Navigation bar tokens repository](https://www.figma.com/design/AGpy41dOJU1kWV2GF22WJX/Navigation---Bar?node-id=16002-35707&t=OaVAEqbyIe9gzFC8-4)
 
#### References added:
- [ ] control-control-item-space-padding-block-top-error-text→space-padding-block-xsmall

#### References renamed:
- [ ] control-control-item-space-padding-block→control-control-item-space-padding-block-default

### Color-repository (Sosh)

References updated:
- [ ] color-repository-secondary-low→core-sosh-color-blueDuck-dark-400
- [ ] color-repository-secondary-medium→core-sosh-color-blueDuck-dark-560

### Color (Sosh)

References updated:
- [ ] color-surface-brand-secondary-light→color-repository-secondary-low
- [ ] color-content-brand-secondary-light→color-repository-secondary-low
- [ ] color-border-brand-secondary-light→color-repository-secondary-low
- [ ] color-action-pressed-light→color-repository-secondary-low
- [ ] color-action-loading-light→color-repository-secondary-low
- [ ] color-action-highlighted-light→color-repository-secondary-low
 
References renamed:
- [ ] color-repository-primary-default→color-repository-primary-medium
- [ ] color-repository-secondary-default→color-repository-secondary-medium
- [ ] color-repository-tertiary-default→color-repository-tertiary-medium
- [ ] color-repository-positivie-default→color-repository-positivie-medium
- [ ] color-repository-info-default→color-repository-info-medium
- [ ] color-repository-warning-default→color-repository-warning-medium
- [ ] color-repository-negative-default→color-repository-negative-medium
- [ ] color-repository-accent-default→color-repository-accent-medium

### Effect

References added:
- [ ] effect-blur-navigation-bar→ouds-core-effect-blur-160→⚠️ WIP: Navigation bar
- [ ] opacity-medium→ouds-core-opacity-560

References renamed:
- [ ] opacity-weaker→opacity-weakest
- [ ] opacity-weak→opacity-weaker
- [ ] opacity-medium→opacity-weak

### Control-switch (Sosh)

References updated:
- [ ] control-switch-color-track-selected-light→color-repository-secondary-low
- [ ] control-switch-color-track-selected-dark→color-repository-secondary-lower
- [ ] control-switch-color-track-selected-interaction-light→color-repository-secondary-medium
- [ ] control-switch-color-track-selected-interaction-dark→color-repository-secondary-lowest
 
### Control-control-item

References added:
- [ ] control-control-item-space-padding-inline-error-icon→dimension-space-padding-inline-4xsmall
- [ ] control-control-item-size-error-icon→ouds-size-icon-with-label-large- size-xsmall

Navigation-bar→⚠️ WIP: Navigation bar
Numerous references were added to meet the needs of the component's creation.

 
# Component

### References added:

- [ ] comp-changelog-dialog-alert-message→1.0.0
- [ ] comp-changelog-dialog-inline-alert→1.0.0

The changelog file is available here:
[https://www.figma.com/design/Co2t6wHMf4GB9NJVGs2Hes/-OUDS-Lib--Design-tokens?node-id=11683-21997&am…](https://www.figma.com/design/Co2t6wHMf4GB9NJVGs2Hes/-OUDS-Lib--Design-tokens?node-id=11683-21997&t=oIMR9q7leDxw9T26-4)

### References added

- [ ] comp-changelog-navigation-bar→1.0.0

## References updated

- [ ] comp-changelog-input-checkbox→2.4.0
- [ ] comp-changelog-input-radiobutton→1.4.0
- [ ] comp-changelog-input-switch→1.5.0

# Core OUDS

#### References added:
- [ ] dimension-1050→84

### Effect
References added:
- [ ] effect-blur-160→16→ ⚠️ WIP: Navigation bar

### Color-opacity
References added:
- [ ] color-opacity-gray-light-80-800→#F4F4F4 (80%)→⚠️ WIP: Navigation bar
- [ ] color-opacity-gray-dark-800-800→#1F1F1F (80%)→⚠️ WIP: Navigation bar

# Core Wireframe 1.3

### Color-opacity

References added:
- [ ] color-opacity-gray-light-80-800→#F7F8FA (80%)→⚠️ WIP: Navigation bar
- [ ] color-opacity-gray-dark-800-800→#181C26 (80%)→⚠️ WIP: Navigation bar

**Comment by vprothais** [comment]
Can be tested here : https://deploy-preview-3233--boosted.netlify.app/
